miner/bid_simulator.go

```diff
@@ -692,5 +742,261 @@ func (r *BidRuntime) commitTransaction(chain *core.BlockChain, chainConfig *para
 		env.receipts = append(env.receipts, receipt)
 	}
 
+	r.env.tcount++
+
 	return nil
 }
+
+// fillTransactions fill pending transactions from the txpool into the bid.
+func (r *BidRuntime) fillTransactions(
```

Comment:
It that possible that we dont have to copy so many code?

Reply:
fixed